src/components/navbar/index.jsx

```
@@ -75,19 +75,19 @@ const Navbar = () => {
       </div>
       <div className="flex-center gap-2 relative">
         <CiSettings
           size={23}
           className="hover:opacity-60"
-          onClick={() => setShowSetting(true)}
+        // onClick={() => setShowSetting(true)}
         />
         <div
           className="flex-center gap-2"
           onClick={() => setIsActiveDropDown((prev) => !prev)}
         >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)